core/cloudflow-operator/src/main/scala/cloudflow/operator/runner/Runner.scala

```diff
@@ -263,6 +281,7 @@ final case class PodsConfig(pods: Map[String, PodConfig] = Map()) {
 }
 
 final case class PodConfig(
+    labels: Option[Map[String, String]] = None,
     containers: Map[String, ContainerConfig]
 )
```

Comment:
I would prefer:
```
final case class PodConfig(
    containers: Map[String, ContainerConfig],
    labels: Map[String, String] = Map.empty[String, String]
)

```

Reply:
Done

Reply:
Please do empty map instead of Option[Map]?

Reply:
(facepalm) I saw the order but not the Option.

Reply:
Done

Reply:
Actually I put it back Option otherwise it will be required that `labels` when creating defining the config file.